src/test/java/eu/europeana/metis/sandbox/test/utils/S3ContainerInitializerIT.java

```diff
@@ -0,0 +1,66 @@
+package eu.europeana.metis.sandbox.test.utils;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.containers.localstack.LocalStackContainer.Service.S3;
+
+public class S3ContainerInitializerIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(S3ContainerInitializerIT.class);
+    static final LocalStackContainer s3Container;
+    static final AmazonS3 s3Client;
+    public static final String S3_VERSION = "localstack/localstack:1.1.0";
```

Comment:
We checked this version with Simon, for S3 compatible testcontainer and is good.

Reply:
just add
```
private static final LocalStackContainer s3Container;
private static final AmazonS3 s3Client;
```

Reply:
Done